README.md

```diff
@@ -510,7 +510,7 @@ can install some projects that depend on MATLAB, in particular:
  * the native MATLAB bindings of YARP, contained in the [yarp-matlab-bindings](https://github.com/robotology-playground/yarp-matlab-bindings/) repository,
  * the MATLAB bindings of qpOASES, contained in the [robotology-dependencies/qpOASES](https://github.com/robotology-dependencies/qpOASES) fork,
  * The [WB-Toolbox](https://github.com/robotology/WB-Toolbox) Simulink toolbox,
- * The [whole-body-controllers](https://github.com/robotology/whole-body-controllers) Simulink-based balancing controllers.
+ * The [whole-body-controllers](https://github.com/robotology/whole-body-controllers) Simulink-based balancing controllers. Note that whole-body-controllers can be installed and compiled also without Matlab, but its functionalities are reduced.
```

Comment:
```suggestion
 * The [whole-body-controllers](https://github.com/robotology/whole-body-controllers) Simulink-based balancing controllers. Note that whole-body-controllers can be installed and compiled also without MATLAB, but its functionalities are reduced.
```
According to Wikipedia and the official MathWorks communication material and documentation, MATLAB is always capitalized.